tools/depends/target/config.site.in

```diff
@@ -103,9 +92,12 @@ if test "@platform_os@" = "darwin_embedded"; then
   fi
 fi
 
-# Python3 build for appletvos
 if test "@target_platform@" = "appletvos"; then
+  # Python3 build
   ac_cv_func_sigaltstack=no
+  # gnutls build
+  ac_cv_func_vfork_works=no
+  ac_cv_func_fork=no
```

Comment:
Doesn't that apply for binary add-ons too?

Reply:
You mean also set this in `config-binaddons.site.in`. It wasn't required before now.